deployment/terraform/modules/haystack-apps/kubernetes/alerting/ad-manager/templates/deployment_yaml.tpl

```diff
@@ -34,28 +34,29 @@ spec:
             cpu: ${cpu_request}
             memory: ${memory_request}Mi
         env:
-        - name: "HAYSTACK_OVERRIDES_CONFIG_PATH"
+        - name: "AA_OVERRIDES_CONFIG_PATH"
           value: "/config/ad-manager.conf"
-        - name: "HAYSTACK_GRAPHITE_HOST"
+        - name: "AA_GRAPHITE_HOST"
           value: "${graphite_host}"
-        - name: "HAYSTACK_GRAPHITE_PORT"
+        - name: "AA_GRAPHITE_PORT"
           value: "${graphite_port}"
-        - name: "HAYSTACK_GRAPHITE_ENABLED"
+        - name: "AA_GRAPHITE_ENABLED"
           value: "${graphite_enabled}"
         - name: "JAVA_XMS"
           value: "${jvm_memory_limit}m"
         - name: "JAVA_XMX"
           value: "${jvm_memory_limit}m"
         ${env_vars}
-        livenessProbe:
-          exec:
-            command:
-            - grep
-            - "true"
-            - /app/isHealthy
-          initialDelaySeconds: 30
-          periodSeconds: 5
-          failureThreshold: 6
+        # FIXME Reinstate after removing haystack-commons dependency
```

Comment:
Is it that we need to remove haystack-commons or is it that we need to reinstate the healthcheck?

Reply:
Need to reinstate the health check